server/chat-commands/admin.ts

```diff
@@ -1102,3 +1121,14 @@ export const commands: ChatCommands = {
 		`[player] must be a username or number, [pokemon] must be species name or party slot number (not nickname), [move] must be move name.`,
 	],
 };
+
+export const pages: PageTable = {
+	bot(args, user) {
+		const [bot, pageid] = args;
+		const targetBot = Users.get(bot);
+		if (!targetBot) {
+			return `<div class="pad"><h2>The bot "${bot}" is not available.</h2></div>`;
+		}
+		targetBot.send(`|requestpage|${user.name}|${pageid}`);
```

Comment:
How is this going to affect regular users? That's probably worth a check?

Reply:
I think it adds some red text to Lobby, which is non-ideal.

Reply:
I'll write a check in.

Reply:
(looks like it just adds some ``code`` text to lobby)

Reply:
It should maybe be a PM?

Reply:
Yeah, a PM from the user; that would probably make it least confusing if a user accidentally gets it.

Reply:
A pm seems more confusing? It's a lot more noticeable, not to mention that if you're not in lobby you won't see the `code` message, so it'd be harder to spam that. 
(also |requestpage| seems easier for bots to parse)

Reply:
You can PM `|requestpage|` to the bot.

I don't want to spam users in lobby either way. PMs can be ignored.

Reply:
Do you want both of these to happen, or just one? 
I've written a check in to ensure that the user is a bot in at least one room (which i'm waiting to commit) -  would you prefer it also be a pm'd message? 
(personally i don't think it should be, since it'd be easier to parse for said bot, but that's just me)

Reply:
Checking that the user is a bot in at least one room is a good start, but I think it should also be a PM just in case.

It's still approximately as easy to parse for the bot, but it'd be a lot easier to figure out if someone makes a mistake somewhere.